pkg/cloud/services/networking/port.go

```diff
@@ -89,11 +89,12 @@ func (s *Service) GetOrCreatePort(eventObject runtime.Object, clusterName string
 				MACAddress: ap.MACAddress,
 			})
 		}
-
-		securityGroups = portOpts.SecurityGroups
-
+		securityGroups, err = s.CollectPortSecurityGroups(eventObject, portOpts.SecurityGroups, portOpts.SecurityGroupFilters)
+		if err != nil {
+			return nil, fmt.Errorf("multiple ports found with name \"%s\"", portName)
```

Comment:
should we only log the err itself here?

Reply:
The error message is wrong, will remove the text. 
However, we need to return the error as it tells the absence of the security groups specified in the manifests.